airflow/airflow.go

```diff
@@ -118,14 +126,21 @@ func Init(path, airflowImageName, airflowImageTag string) error {
 		".astro/dag_integrity_exceptions.txt":  "# Add dag files to exempt from parse test below. ex: dags/<test-file>",
 	}
 
-	// Initailize directories
-	if err := initDirs(path, dirs); err != nil {
-		return errors.Wrap(err, "failed to create project directories")
-	}
+	if template != "" {
+		err := templateClient.DownloadAndExtractTemplate(astroTemplateRepoURL, template, path)
```

Comment:
nit: keeping the `templateClient` initialization only when `template != ""` might be a bit better